src/test/java/wolox/training/BookFactory.java

```diff
@@ -0,0 +1,35 @@
+package wolox.training;
+
+import wolox.training.models.Book;
+
+public class BookFactory {
+  public BookFactory() {}
+
+  public Book createTestBook(){
+    Book book = new Book();
+    book.setAuthor("Author");
+    book.setImage("Image");
+    book.setTitle("Title");
+    book.setSubtitle("Subtitle");
+    book.setPublisher("Publisher");
+    book.setYear("2020");
+    book.setPages(20);
+    book.setIsbn("ISBN");
+    book.setGenre("Genre");
+    return book;
+  }
+
+  public String getJsonBook(Book book){
+    return "{\"genre\": \"" + book.getGenre() + "\"," +
```

Comment:
Can you make this object with Jackson? if it's very difficult leave this

Reply:
BookFactory work well with Jackson, UserFactory fails because of the LocalDate attribute. I'll leave userFactory with the ugly string since it has just 3 attributes